### PR TITLE
Update Script.js

### PR DESCRIPTION
**Pull request by ashishkadyan981**

### CODE REVIEW

File: Script.js [hunk 1]
```diff
@@ -12,3 +12,5 @@ document.getElementById("btn").addEventListener("click", getColor);
 //initial call
 getColor();
 console.log("hii this is test for how pull req. works.");
+
+console.log("just only for other tests.");
```
Comment (Ashishkdyn) [inline Script.js:16]: why you insert this, because i inserted console already.

Comment (ashishkadyan981) [reply Script.js:16]: because it's my choice

Comment (Ashishkdyn) [reply Script.js:16]: chal gandu

Comment (Ashishkdyn) [reply Script.js:16]: close kr isko gandu khi ka befaltu ke pulls dalta rhta hai

**Pull request closed without merging**